app/concepcts/reservation_seats/repository.rb

```diff
@@ -15,5 +15,9 @@ def reserved_seats(screening_id)
     def create(seat_id, reservation_id)
       adapter.create(seat_id: seat_id, reservation_id: reservation_id)
     end
+
+    def delete_seats_reservations_by_reservation_id(reservation_id)
```

Comment:
naming is hard ;)
but maybe
```
def delete_by(filter)
  adapter.where(filter).delete_all
end
```

and then when you want to run it you can use
`repo.delete_by(reservation_id: reservation_id)`